lib/upload_system_log.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+#Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+package upload_system_log;
+# Summary:  base class for collecting and uploading system log
+# Maintainer: Yong Sun <yosun@suse.com>
+
+use strict;
+use base "Exporter";
+use Exporter;
+use testapi;
+use utils;
+use base "opensusebasetest";
+
+our @EXPORT = qw(upload_system_logs);
+
+
```

Comment:
Really a small thing, remove duplicate empty line ;-)

Reply:
For perfect, it should be removed :)